File: VB/DXApplication14/Appt.vb
Imports DevExpress.Mvvm
Imports System

Namespace DXApplication14
	Public Class Appt
		Inherits BindableBase


		Protected _Subject As String
		Public Property Subject() As String
			Get
				Return Me._Subject
			End Get
			Set(ByVal value As String)
				Me.SetProperty(Me._Subject, value, "Subject")
			End Set
		End Property


		Protected _Start As DateTime
		Public Property Start() As DateTime
			Get
				Return Me._Start
			End Get
			Set(ByVal value As DateTime)
				Me.SetProperty(Me._Start, value, "Start")
			End Set
		End Property


		Protected _End As DateTime
		Public Property [End]() As DateTime
			Get
				Return Me._End
			End Get
			Set(ByVal value As DateTime)
				Me.SetProperty(Me._End, value, "End")
			End Set
		End Property

		Protected _Description As String
		Public Property Description() As String
			Get
				Return Me._Description
			End Get
			Set(ByVal value As String)
				Me.SetProperty(Me._Description, value, "Description")
			End Set
		End Property


		Protected _Type As Integer
		Public Property Type() As Integer
			Get
				Return Me._Type
			End Get
			Set(ByVal value As Integer)
				Me.SetProperty(Me._Type, value, "Type")
			End Set
		End Property


		Protected _RecurrenceInfo As String
		Public Property RecurrenceInfo() As String
			Get
				Return Me._RecurrenceInfo
			End Get
			Set(ByVal value As String)
				Me.SetProperty(Me._RecurrenceInfo, value, "RecurrenceInfo")
			End Set
		End Property
	End Class
End Namespace

File: CS/DXApplication14/Appt.cs
using DevExpress.Mvvm;
using System;

namespace DXApplication14
{
    public class Appt : BindableBase {


        protected string _Subject;
        public string Subject
        {
            get { return this._Subject; }
            set { this.SetProperty(ref this._Subject, value, "Subject"); }
        }


        protected DateTime _Start;
        public DateTime Start
        {
            get { return this._Start; }
            set { this.SetProperty(ref this._Start, value, "Start"); }
        }


        protected DateTime _End;
        public DateTime End
        {
            get { return this._End; }
            set { this.SetProperty(ref this._End, value, "End"); }
        }

        protected string _Description;
        public string Description
        {
            get { return this._Description; }
            set { this.SetProperty(ref this._Description, value, "Description"); }
        }


        protected int _Type;
        public int Type
        {
            get { return this._Type; }
            set { this.SetProperty(ref this._Type, value, "Type"); }
        }


        protected string _RecurrenceInfo;
        public string RecurrenceInfo
        {
            get { return this._RecurrenceInfo; }
            set { this.SetProperty(ref this._RecurrenceInfo, value, "RecurrenceInfo"); }
        }
    }
}
